guide/src/main/asciidoc/api_specifications.adoc

```diff
@@ -14,17 +14,17 @@ In this case, special attention should be given that the API specifications rema
 [[openapi]]
 === OpenAPI (Swagger)
 
-OpenAPI is the dominating standard for API documentation, having gained wide industry support.
+OpenAPI is the dominating standard for REST API documentation, having gained wide industry support.
 It is based on http://swagger.io/[Swagger^], which has been standardized as https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md[OpenAPI 2.0].
 
 [rule, oas-contra]
 .OpenAPI
 ====
-Specifications of the API SHOULD be provided using https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md[*OpenAPI 2.0*] (aka Swagger) or https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md[*OpenAPI 3.0*].
+Specifications of a new REST API SHOULD be provided using https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md[*OpenAPI 3.0*].
```

Comment:
There is 3.0.4 since October last year. Do we have any recommendations by the way about the patch version to use?